packages/core/src/RenderingEngine/VolumeViewport.ts

```diff
@@ -82,6 +83,13 @@ class VolumeViewport extends BaseVolumeViewport {
     return super.setVolumes(volumeInputArray, immediate, suppressEvents);
   }
 
+  /** Gets the number of slices the volume is broken up into in the camera direction */
+  public getNumberOfSlices(): number {
```

Comment:
This is the implementation used elsewhere, seems like it is reasonable to just call it from here.